infra/eligibility_screener/template/ecs.tf

```diff
@@ -161,6 +185,10 @@ resource "aws_ecs_task_definition" "eligibility-screener-ecs-task-definition" {
         }
       ],
       environment : [
+        {
+          "name" : "NEXT_PUBLIC_DEMO_MODE"
+          "value" : "true"
+        },
```

Comment:
@aplybeah It turns out that we _don't_ want this. See https://github.com/navapbc/wic-mt-demo-project-eligibility-screener/pull/172